android/assets/jsons/Civ V - Gods & Kings/TileResources.json

```diff
@@ -474,7 +457,7 @@
 		"food": 1,
 		"gold": 1,
 		"improvement": "Plantation",
-		"improvementStats": {"food": 1,"gold": 1}
+		"improvementStats": {"food": 1}
```

Comment:
Wiki says this should be +1 Gold (and then +1 Food only with Calendar)?

https://civilization.fandom.com/wiki/Cocoa_(Civ5)

…Hm. Also shouldn't exist without BNW, apparently.

Reply:
It is commented out anyway.